dom/data/data-test.js

```diff
@@ -123,3 +123,28 @@ unit.test('domData should be cleaned up if element is removed from DOM after cal
 
 	checkRemoved();
 });
+
+// MutationObserver is needed for this test
+if(typeof MutationObserver !== 'undefined'){
+	unit.test('domData should count active elements', function (assert){
```

Comment:
This test needs more to check things are working. Specifically:

- If you call set on the same element multiple times, it should only increment the first time.
- If you call delete on the same element multiple times, it should only decrement the first time.